src/test/java/com/github/kaklakariada/fritzbox/mapping/DeserializerTest.java

```diff
@@ -78,4 +80,25 @@ public void parseDeviceListAllTogether() throws IOException {
         new Deserializer().parse(fileContent, DeviceList.class);
     }
 
+    @Test
+    public void parseDeviceList() throws IOException {
+        final String fileContent = Files.readAllLines(Paths.get("src/test/resources/deviceList.xml")).stream()
+                .collect(joining("\n"));
+        new Deserializer().parse(fileContent, DeviceList.class);
+    }
+
+    @Test
+    public void parseSessionInfo() throws IOException {
```

Comment:
[Minor] With my from today PR I just introduced assertJ into the project. Maybe you want to have a look at those Assertion classes which allow fluent-API assertions. BR, Peter.

Reply:
Will have a look, but as merging is not my favourite git command, i would update these tests with my next contribution.

Reply:
Rebasing ist not required. I also don't use it ;)